src/visitors/ispc_rename_visitor.cpp

```diff
@@ -0,0 +1,64 @@
+/*************************************************************************
+ * Copyright (C) 2018-2020 Blue Brain Project
+ *
+ * This file is part of NMODL distributed under the terms of the GNU
+ * Lesser General Public License. See top-level LICENSE file for details.
+ *************************************************************************/
+
+#include "visitors/ispc_rename_visitor.hpp"
+
+#include "ast/all.hpp"
+#include "parser/c11_driver.hpp"
+#include "visitors/visitor_utils.hpp"
+
+
```

Comment:
one general question here: why we re-use RenameVisitor here? i.e. I can following possibilities:
* In this visitor only find problematic variable and then invoke RenameVisitor
* Look into symbol table for all variable names, find out problematic ones and then invoke RenameVisitor

Reply:
I had exactly the same question...

Reply:
The reason why I didn't reuse the `RenameVisitor` is that this would mean two passes on the same variables. One to find which need changing and one to change them with the `RenameVisitor`. What do you think about that?

Reply:
@cattabiani There is already a conversation here regarding editing the `RenameVisitor` and the separate `IspcRenameVisitor`. I think that creating a new visitor is okay to avoid having making the `RenameVisitor` more complex for something that is related only for the `ispc` backend. If you guys think otherwise I can change it

Reply:
ok!